tests/lib/bin/retry

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+COMMAND="${1:-true}"
```

Comment:
Could you please replace this with rudimentary parsing of `--max-attempts` and `--sleep-time` and shift the parsed options away. Alternatively YAGNI and don't even have this feature.

Reply:
done